taf/TAF_Runner.py

```diff
@@ -1016,6 +1015,7 @@ def __create_formated_lkup(self, filetyp: str, fl: str, fl2: str, rpt_out: str,
 
             rstr = hash(time.time())
             df_sp.createOrReplaceTempView(f"pgm_audt_cnt_lkp_{rstr}")
+            print(df_sp)
```

Comment:
Was this a debugging statement? Maybe it could be removed or changed to logger.debug?

Reply:
Not needed. Debugging locally can make a mess of things. Supposedly, there is a way to step debug for remote Databricks but haven't figured it out yet.